shell/platform/fuchsia/flutter/surface.cc

```diff
@@ -26,8 +26,10 @@ bool Surface::IsValid() {
 std::unique_ptr<flutter::SurfaceFrame> Surface::AcquireFrame(
     const SkISize& size) {
   return std::make_unique<flutter::SurfaceFrame>(
-      nullptr, [](const flutter::SurfaceFrame& surface_frame,
-                  SkCanvas* canvas) { return true; });
+      nullptr, true,
```

Comment:
Need to verify if this is the right value for surface_supports_readback for Fuchsia surfaces.

Reply:
Trivially, this routine is returning a dummy SurfaceFrame (note that the surface itself is nullptr) so the value doesn't really matter.